crates/cxx-qt-gen/src/generator/structuring/qobject.rs

```diff
@@ -44,7 +44,7 @@ impl<'a> StructuredQObject<'a> {
             .map(|method| &method.name)
             .find(|name| name.rust_unqualified() == id)
             .cloned()
-            .ok_or_else(|| Error::new_spanned(id, format!("Method with name '{id}' not found!")))
+            .ok_or_else(|| Error::new_spanned(id, format!("Method with name `{id}` not found!")))
```

Comment:
Hm, this is pretty much a duplicate with line 56 apart from `Method`->`Signal`.

Maybe put this in a shared function:
```rust
fn err_member_not_found(kind: &str, id: &str) -> Error
```